pyspark_ai/pyspark_ai.py

```diff
@@ -60,6 +61,7 @@ def __init__(
         cache_file_format: str = "json",
         cache_file_location: Optional[str] = None,
         vector_store_dir: Optional[str] = None,
+        vector_store_max_size: Optional[float] = 1e6,
```

Comment:
Shall we make it an Int? What's the unit here, KB?

Reply:
Or let's keep it as float, but the unit can be GB or MB so that is it might be easier to use.